escrow/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "hardhat-tutorial-escrow",
+  "version": "1.0.0",
+  "main": "index.js",
+  "author": "Lauro Gripa",
+  "license": "MIT",
+  "dependencies": {
+    "hardhat": "^2.6.8"
+  },
+  "scripts": {
+    "test": "hardhat test test/Escrow.js",
+    "test-mandala": "hardhat test test/Escrow.js --network mandala",
+    "test-mandala:pubDev": "hardhat test test/Escrow.js --network mandalaPubDev",
+    "test-mandala:CI": "hardhat test --network mandalaCI",
+    "build": "hardhat compile",
+    "deploy": "hardhat run scripts/deploy.js",
+    "deploy-mandala": "hardhat run scripts/deploy.js --network mandala",
+    "deploy-mandala:pubDev": "hardhat run scripts/deploy.js --network mandalaPubDev",
+    "deploy-mandala:CI": "hardhat run scripts/deploy.js --network mandalaCI",
+    "loop": "hardhat run test/loop.js --network mandala",
+    "loop:CI": "hardhat run test/loop.js --network mandalaCI"
+  },
+  "devDependencies": {
+    "@acala-network/contracts": "^3.0.0",
+    "@acala-network/eth-providers": "^2.1.7",
+    "@nomiclabs/hardhat-ethers": "^2.0.2",
+    "@nomiclabs/hardhat-waffle": "^2.0.1",
+    "@openzeppelin/contracts": "^4.4.0",
```

Comment:
I think this can be removed. I don't used it directly, it's used through `@acala-network/contracts`